spinalcordtoolbox/reports/qc.py

```diff
@@ -776,18 +776,24 @@ def qcslice_layout(x): return x.mosaic()[:2]
         p_resample = p_resample_default
     elif p_resample == 0:   # If user specified `-resample 0`, turn off resampling
         p_resample = None
-    qcslice = SliceSubtype(im_list, p_resample=p_resample)
-    qc_report = QcReport(fname_in1, process, args, plane, path_qc, dpi, dataset, subject)
 
-    QcImage(
+    qcslice = object.__new__(SliceSubtype)
```

Comment:
For my own understanding... is there a difference between the `object.__new__.Slice` syntax (used here) and the `Slice()` syntax (used in the test refactors)?

Reply:
Oh, good catch!

In general, yes, there's an important difference. The `Slice()` syntax implicitly calls `Slice.__init__`, whereas `object.__new__(Slice)` doesn't.

In this case, the `Sagittal.__init__` and `Slice.__init__` no longer exist, so they have the same effect. But during the refactor, I hadn't gotten rid of `Slice.__init__` yet, so I had to split the object creation into `object.__new__` and `x.__init__`.

Reply:
Aha, gotcha!